### PR TITLE
ASMA_0009

### PR DESCRIPTION
**Pull request by faraiwande**
I understand that this is part of continuous deployment, but what is confusing me a bit is that we have set this to deploy on a specific branch & in this case 0009 (Module 9). We actually want this to be on the main branch that will have gone through some sort of code review or as in the DevOps ways of working as long as the Tests Pass we are good to go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,3 +131,13 @@ To update the app you need to run the following commands:
 3. Then trigger Azure to pull the updated image from Dock by making a post request to the webhook link which you can find on the App Service on the Deployment Centre Tab. 
     
 
+## Automate Azure Deployment
+
+To automatically deploy the application to Azure you will need to your Docker Credentials & the webhook link of your App Service as Secrets in your repository.
```
Comment (Jonesey13) [inline README.md:136]:
```suggestion
To automatically deploy the application to Azure you will need to add your Docker Credentials & the webhook link of your App Service as GitHub Actions Secrets in your repository. 
```

File: .github/workflows/my_ci_pipeline.yaml [hunk 2]
```diff
@@ -8,9 +8,28 @@ on:
...
     - run: docker build --target test --tag todo_app:test -f dockerfiles/Dockerfile.todo_app .
-    - run: docker run todo_app:test
\ No newline at end of file
+    - run: docker run todo_app:test
+
+  deploy: 
+    name: Docker Hub Push & Triggers App Service Deploy
+    runs-on: ubuntu-latest
+    needs: build 
+    if: ${{github.event_name=='push' && github.ref=='refs/heads/ASAMA_0009'}}
```
Comment (Jonesey13) [inline .github/workflows/my_ci_pipeline.yaml:22]: Like you mentioned you'll want to change the branch name back to `main` after merging the PR

**Review by Jonesey13 — APPROVED** [review]
Looks great to me! Please see the couple of comments.

File: .github/workflows/my_ci_pipeline.yaml (hunk 2 above)
Comment (faraiwande) [reply .github/workflows/my_ci_pipeline.yaml:22]: Thanks as I was a bit confused inititally looking at it

**Pull request merged**